perfkitbenchmarker/aws/aws_network.py

```diff
@@ -101,12 +101,27 @@ def _Create(self):
         'create-vpc',
         '--region=%s' % self.region,
         '--cidr-block=10.0.0.0/16']
-    stdout, _ = vm_util.IssueRetryableCommand(create_cmd)
+    stdout, _, _ = vm_util.IssueCommand(create_cmd)
     response = json.loads(stdout)
     self.id = response['Vpc']['VpcId']
     self._EnableDnsHostnames()
     util.AddDefaultTags(self.id, self.region)
 
+  def _Exists(self):
+    """Returns true if the VPC exists."""
+    describe_cmd = util.AWS_PREFIX + [
+        'ec2',
+        'describe-vpcs',
+        '--region=%s' % self.region,
+        '--filter=Name=vpc-id,Values=%s' % self.id]
+    stdout, _ = vm_util.IssueRetryableCommand(describe_cmd)
+    response = json.loads(stdout)
+    vpcs = response['Vpcs']
+    if not vpcs:
```

Comment:
As above, consider asserting the length of the VPCs (in case an API changes).  Nit: `return len(vpcs) > 0` would be a little clearer, IMO.